copulas/univariate/base.py

```diff
@@ -180,14 +180,18 @@ def to_dict(self):
         """Returns parameters to replicate the distribution."""
         result = {
             'type': get_qualified_name(self),
-            'fitted': self.fitted,
+            'fitted': self.fitted
         }
 
         if not self.fitted:
             return result
 
         if get_qualified_name(self) == get_qualified_name(Univariate):
-            result['instance_type'] = get_qualified_name(self._instance)
+            if self.constant_value is not None:
```

Comment:
This modifies the univariate base class so it can be saved/loaded even if the value is constant.